db/migrate/20201118111330_add_timestamps_to_job_alert_feedbacks.rb

```diff
@@ -0,0 +1,7 @@
+class AddTimestampsToJobAlertFeedbacks < ActiveRecord::Migration[6.0]
+  def change
+    add_timestamps :job_alert_feedbacks, default: DateTime.new(2020, 1, 1)
```

Comment:
@david-mears-dfe what would you think of setting this to the launch date of the feedback feature in October instead? Might make the URs' feedback dashboard easier to navigate.

Reply:
If it makes the dashboard easier to use, then yes let's.